hillel-expense-tracker-service/src/test/java/ua/ithillel/expensetracker/util/Base64ConverterTest.java

```diff
@@ -0,0 +1,68 @@
+package ua.ithillel.expensetracker.util;
+
+import lombok.SneakyThrows;
+import org.junit.jupiter.api.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class Base64ConverterTest {
+    private final Base64Converter base64Converter = new Base64Converter();
+
+    @Test
+    void encodeToStringTest_returnsString() throws IOException {
+        String testString = "testString\n";
+        String expectedResult = "dGVzdFN0cmluZwo=";
+        InputStream in = new ByteArrayInputStream(testString.getBytes());
+
+        String result = base64Converter.encodeToString(in);
+
+        assertNotNull(result);
+        assertFalse(result.isEmpty());
```

Comment:
These two seem unnecessary, as these checks are basically performed in `assertEquals(expectedResult, result);`, what do you think?